File: wlr/OGUR/OGUR/Entities/Floor.cs
using SPX.Entities;
using OGUR.Management;
using SPX.Core;
using OGUR.Entities;
using System;
using System.Runtime.Serialization;

namespace OGUR.Entities
{
    [Serializable()]
    public class Floor : Entity
    {
        public Floor(SerializationInfo info, StreamingContext context):base(info, context){}
        public Floor(Point2 location)
        {
            Initialize(location, SpriteType.FLOOR, OGUR.EntityType.FLOOR,ZDepth.Floor);
        }
    }
}

File: proj/src/aigilas/entities/Floor.java
package aigilas.entities;

import aigilas.management.SpriteType;
import spx.bridge.DrawDepth;
import spx.bridge.EntityType;
import spx.core.Point2;
import spx.entities.Entity;

public class Floor extends Entity {
    public Floor(Point2 location) {
        Initialize(location, SpriteType.FLOOR, EntityType.FLOOR, DrawDepth.Floor);
    }
}
